web/static/js/components/idea_edit_form.jsx

```diff
@@ -63,6 +71,16 @@ class IdeaEditForm extends Component {
             <option key={category} value={category}>{category}</option>
           ))}
         </select>}
+        {stage === "action-items" && <select
```

Comment:
a couple things here. The first is that we should attempt to re-use the `SelectDropdown` component that we introduced in the "action items assignment" story. The second is that we should leverage the stage constants found in the configs/stages.js file, both here (line 74) and [up above](https://github.com/stride-nyc/remote_retro/blob/96139549351ce2c9b93ced38eedf36c63e4ef683/web/static/js/components/idea_edit_form.jsx#L65)

Reply:
and in fact, we should also use the `SelectDropdown` [up above as well](https://github.com/stride-nyc/remote_retro/blob/96139549351ce2c9b93ced38eedf36c63e4ef683/web/static/js/components/idea_edit_form.jsx#L64-L73).